source/MaterialXCore/Node.cpp

```diff
@@ -376,4 +376,18 @@ string GraphElement::asStringDot() const
     return dot;
 }
 
+//
+// NodeGraph methods
+//
+
+InterfaceElementPtr NodeGraph::getImplementation() const
```

Comment:
Add in a utility for this since there is none at this level. Also follows from spec conformance discussion that implementations for nodegraphs don't require language or target.